actors/badge/src/main/java/org/sunbird/badge/util/BadgeAssertionValidator.java

```diff
@@ -0,0 +1,73 @@
+package org.sunbird.badge.util;
+
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.badge.model.BadgeClassExtension;
+import org.sunbird.badge.service.BadgeClassExtensionService;
+import org.sunbird.badge.service.impl.BadgeClassExtensionServiceImpl;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.BadgingJsonKey;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.responsecode.ResponseCode;
+
+/**
+ * Class to provide validation for Assertion of badge
+ *
+ * @author arvind
+ */
+public class BadgeAssertionValidator {
+
+  private static BadgeClassExtensionService badgeClassExtensionService =
+      new BadgeClassExtensionServiceImpl();
+  private static CassandraOperation cassandraOperation;
+  private static final String USER_TABLE_NAME = "user";
```

Comment:
take table name and keyspace from some common variable , which is used globally. other wise if any change will happen other place this will fail.